admin_guide/limits.adoc

```diff
@@ -87,7 +87,36 @@ specified.
 <13> The default amount of memory that a container will request to use if not specified.
 <14> The maximum amount of CPU burst that a container can make as a ratio of its limit over request.
 ====
-// end::admin_limits_sample_definition[]
+
+.OpenShift Limit Range Object Definition
+====
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "LimitRange"
+metadata:
+  name: "openshift-resource-limits"
+spec:
+  limits:
+    -
```

Comment:
Sorry, I was wrong. I thought I tried it before and it didn't work in this case. I tried now with:

    limits:
      - type: openshift.io/Image
        max:
          storage: 1Gi
      - type: openshift.io/ImageStream
        max:
          openshift.io/image-tags:
          openshift.io/images: 30

And `oc create -f ...` consumed it without a problem. The thing is that giving this to a yaml validator, it will indent the **object** aka **associative array** in the same way you see here. Moreover with *squashed* lines the `ascii_binder` will render everything following the dash in red making it look like *this is invalid syntax*.

I can squash it if you want but I'd have to remove the leading `[source,yaml]` line.